src/Paramore.Brighter/IAmACommandProcessor.cs

```diff
@@ -307,13 +307,13 @@ Task ClearOutboxAsync(
         /// Intended for use with the Outbox pattern: http://gistlabs.com/2014/05/the-outbox/ <see cref="DepositPostBoxAsync"/>
         /// </summary>
         /// <param name="amountToClear">The maximum number to clear.</param>
-        /// <param name="minimumAge">The minimum age to clear in milliseconds.</param>
+        /// <param name="minimumAge">The minimum age to clear (Default 5 second).</param>
         /// <param name="useBulk">Use the bulk send on the producer.</param>
         /// <param name="requestContext">The context of the request; if null we will start one via a <see cref="RequestContextFactory"/> </param>
         /// <param name="args">For transports or outboxes that require additional parameters such as topic, provide an optional arg</param>
         void ClearOutstandingFromOutbox(
             int amountToClear = 100, 
-            int minimumAge = 5000, 
+            TimeSpan? minimumAge = null, 
```

Comment:
Should we check for null and set to 5000ms to preserve behavior

Reply:
I am already doing that in the implementation, I also updated the comment to state the default is 5 seconds, although I don't think this belongs on the interface (the comment about the implementation default of 5 second)